pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: fl_chart
 description: A highly customizable Flutter chart library that supports Line Chart, Bar Chart, Pie Chart, Scatter Chart, and Radar Chart.
-version: 0.66.2
+version: 0.67.0
```

Comment:
You should not upgrade the version. Please read the [contributing](https://github.com/imaNNeo/fl_chart/blob/main/CONTRIBUTING.md) guidelines carefully

Reply:
Done